.github/workflows/build-test-windows.yml

```diff
@@ -35,21 +35,59 @@ jobs:
         include:
           - repository: pytorch/tensorrt
             pre-script: packaging/pre_build_script_windows.sh
+            env-script: packaging/vc_env_helper.bat
+            wheel-build-params: "--release"
```

Comment:
This could affect package naming on the nightly index [here](https://download.pytorch.org/whl/nightly/torch-tensorrt/)https://[download.pytorch.org/whl/nightly/torch-tensorrt/](https://download.pytorch.org/whl/nightly/torch-tensorrt/), but we may want it to source the version from the environment on `main`, to include the date instead of the commit hash. Is this flag necessary here?

Reply:
Yes. **Without** this flag, the wheel is named like `torch_tensorrt-2.4.0.dev0+243592a-cp38-cp38-win_amd64.whl`. **With** this flag, the wheel is named like `torch_tensorrt-2.4.0.dev20240506+cu124-cp38-cp38-win_amd64.whl`. I guess the latter is what you want.